backend/eAkademik.API/Services/UserService.cs

```diff
@@ -52,4 +53,28 @@ public async Task EditUser(UserViewModel user)
 
         await _context.SaveChangesAsync();
     }
+
+    public async Task<User> CreateUser(string firstName, string lastName, string email, string password)
+    {
+        if (!new EmailAddressAttribute().IsValid(email))
+            throw new Exception("E-mail adress is incorrect");
+
+        var sameUser = await _context.Users.SingleOrDefaultAsync(x => x.Email == email);
+        if (!(sameUser is null))
```

Comment:
zmieniłbym na 
`sameUser != null` lub `sameUser is not null`
poprawi czytelności